src/python/twitter/common/fs/hdfs.py

```diff
@@ -52,10 +57,14 @@ def config(self):
     return self._config
 
   def _call(self, cmd, *args, **kwargs):
-    """Runs hdfs fs command  with the given command and args.
+    """Runs fs command with the given command and args.
     Checks the result of the call by default but this can be disabled with check=False.
     """
-    cmd = ['hdfs', '--config', self._config, 'dfs', cmd] + list(args)
+    self._hadoop_command = 'hdfs'
+    if self.use_hadoop_v1 or self._cmd_class.execute_suppress_stdout_stderr('hdfs') != 0:
```

Comment:
How are you trying to use this, can you provide a bit more surrounding code?

`use_hadoop_v1` should be passed in to the `HDFSHelper` if you want to set it to a non-default value.  The default is `False`.

This variable is only used in the initializer to set `self.cli_command` to the correct Hadoop command to be used later in operations.